<launch>
    <arg name="name" default="front"/>
    <arg name="mode" default="distance"/>

    <node pkg="single_beam_sonar" type="echosounder.py" name="sonar_driver">
        <param name="name" value="front"/>
        <param name="usb_port" value="/dev/ttyUSB0" />
        <param name="frequency" value="10" />
        <param name="frame" value="odom" />
        <param name="fov" value="0.3" />
        <param name="min_range" value="0.5" />
        <param name="max_range" value="50.0" />
        <param name="mode" value="$(arg mode)"/>
    </node>
</launch>